### PR TITLE
Extra containerized nccl tests

### PR DESCRIPTION
**Pull request by verdimrc**
*Issue #, if available:* N/A

*Description of changes:* sample .sbatch scripts to run nccl tests under containers. Two variants: native implementation, and a pure pytorch-based (that some of our customers have been using for benchmarking).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: micro-benchmarks/nccl-tests/slurm/nccl-tests-container.sbatch
Comment (mhuguesaws) [inline micro-benchmarks/nccl-tests/slurm/nccl-tests-container.sbatch:1]: This is a duplicate of https://github.com/aws-samples/awsome-distributed-training/blob/main/micro-benchmarks/nccl-tests/slurm/nccl-tests.sbatch

File: micro-benchmarks/nccl-tests/slurm/pt-all-reduce-container.sbatch [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    #--tee 3
+)
+
+## Pre-requisite: ${FSX_MOUNT} (this is a path inside the container) must contains the .py script
+## from https://github.com/stas00/ml-engineering/blob/master/network/benchmarks/all_reduce_bench.py
+##
+## Sample command to download:
+##
+##     curl -LO https://raw.githubusercontent.com/stas00/ml-engineering/master/network/benchmarks/all_reduce_bench.py
+srun -l --wait 300 "${ARGS[@]}" torchrun "${TORCHRUN_ARGS[@]}" ${FSX_MOUNT}/all_reduce_bench.py
```
Comment (mhuguesaws) [inline micro-benchmarks/nccl-tests/slurm/pt-all-reduce-container.sbatch:69]: Missing the file.

**Review by mhuguesaws — COMMENTED** [review]
Left comments

File: micro-benchmarks/nccl-tests/slurm/pt-all-reduce-container.sbatch (hunk 1 above)
Comment (verdimrc) [reply micro-benchmarks/nccl-tests/slurm/pt-all-reduce-container.sbatch:69]: Intentional to let user to pull this file from a separate repo.

**Comment by verdimrc** [comment]
To rebase once #314 merged.

**Pull request closed without merging**